<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="27727" started-at="2017-02-19T19:06:26Z" finished-at="2017-02-19T19:06:53Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="27727" started-at="2017-02-19T19:06:26Z" finished-at="2017-02-19T19:06:53Z">
      <class name="TestCases.TestNG_Screenshots">
        <test-method status="PASS" signature="setUp()[pri:0, instance:TestCases.TestNG_Screenshots@77caeb3e]" name="setUp" is-config="true" duration-ms="17253" started-at="2017-02-20T00:36:26Z" finished-at="2017-02-20T00:36:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="test1_invalidCredentials()[pri:0, instance:TestCases.TestNG_Screenshots@77caeb3e]" name="test1_invalidCredentials" duration-ms="7724" started-at="2017-02-20T00:36:43Z" finished-at="2017-02-20T00:36:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at TestCases.TestNG_Screenshots.test1_invalidCredentials(TestNG_Screenshots.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1_invalidCredentials -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:TestCases.TestNG_Screenshots@77caeb3e]" name="tearDown" is-config="true" duration-ms="2716" started-at="2017-02-20T00:36:51Z" finished-at="2017-02-20T00:36:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test1_invalidCredentials status=FAILURE method=TestNG_Screenshots.test1_invalidCredentials()[pri:0, instance:TestCases.TestNG_Screenshots@77caeb3e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCases.TestNG_Screenshots -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
